crates/ruff/src/rules/flake8_unused_arguments/rules/unused_arguments.rs

```diff
@@ -90,15 +90,15 @@ impl Violation for UnusedFunctionArgument {
 ///
 /// ## Example
 /// ```python
-/// class MyClass:
-///     def my_method(self, arg1, arg2):
+/// class Class:
```

Comment:
It seems to me that calling the class `Class` is not right, because there are `class` directive